docs/contributing/DEVELOPING.md

```diff
@@ -144,19 +150,43 @@ query SimpleUnionQuery_only {
 }
 ```
 
-To update the file `performance.json`, with the results of the performance test, run `yarn performance -u`
+**Saving metrics**
+With the option `-u` a file `performance.json` will be generated, that can be used to compare metrics between runs. This is only available for database query benchmark.
 
-#### Running with Cypher
+**Running Cypher queries**
 
-The performance tests can also run raw Cypher, to enable it, run `yarn performance --cypher`. Cypher queries must be located at `tests/performance/cypher`
+The performance tests can also run raw Cypher, to enable it, run `yarn performance --cypher`. Cypher queries must be located at `tests/performance/databaseQuery/cypher`. This allows for comparison between GraphQL and an alternative Cypher query.
 
-#### Running for markdown
+**Generating markdown**
 
-With the option `--markdown` the output will be formatted in markdown.
+With the option `--markdown` the output will be formatted in markdown instead of the CLI. This is only available for database query benchmark.
 
 #### Schema Generation
 
-Running `yarn performance --schema` will run instead the schema generation performance test.
+This benchmark runs the schema generation for a large GraphQL Schema and outputs the time that took to generate it. No database needed.
+
+To execute this, run `yarn performance --schema`.
+
+**Subgraph Schema**
+Alternatively, the schema can be run ready for subgraph by executing `yarn performance --subgraph-schema`.
+
+#### Translation
+
+This benchmark will measure the time it takes to translate a certain GraphQL query (without hitting the database). All of the queries in the `graphql` folder will be used, along with the queries located in `translation/graphql` `_skip` and `_only` can be used to limit how many queries are run.
+
+This can be run with `yarn performance --translation`.
+
+Note that the output will be rounded to milliseconds.
+
+**Change test runs**
+By default, each query will be translated 100 times, and the total time will be shown.
+
+With the argument `--runs` you can change how many runs to do per query (e.g. `yarn performance --translation --runs 1000`).
```

Comment:
In this line you use "argument" while in others you use "option" or "flag", we should probably use consistent language

Reply:
I will change it so it is option, and flag for things that are boolean